boogiestats/boogie_api/models.py

```diff
@@ -185,8 +175,6 @@ def update_search_cache(self, redis_connection: Optional[Redis] = None) -> bool:
             r.hset(f"song:{self.hash}", mapping={k: v for k, v in chart_info.items() if k in fields})
             return True
 
-        return False
```

Comment:
should be restored, it's an artifact of an intermediate implementation that has been since dropped